index.html

```diff
@@ -34,31 +34,244 @@
+        </div>
+    </section>
+    <!-- ─── End Headline Section ────────────────────────────────────── -->
+
+    <!-- ─── Portfolio Works ─────────────────────────────────────────── -->
+    <section class="works">
+        <div class="grid-container">
+            <!-- ─── Works Project 1 ─────────────────────────── -->
+            <div class="works-container">
+                <div class="banner">
+                    <img src="./images/Portfolio2.png" class="project-img" alt="project image">
+                </div>
+                <div class="project-content">
+                    <div class="project-title">
+                        <h2>Tonic</h2>
+                    </div>
+                    <div class="outline">
+                        <li class="banner-list">Canopy</li>
+                        <span class="dot"></span>
+                        <li class="banner-list dev">Back End Dev</li>
+                        <span class="dot"></span>
+                        <li class="banner-list">2015</li>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="tags">
+                        <ul>
+                            <li class="skill">html</li>
+                            <li class="skill">css</li>
+                            <li class="skill">javascript</li>
+                        </ul>
+                    </div>
+                    <div class="action">
+                        <button class="btn">See Project</button>
+                    </div>
+                </div>
+            </div>
+
+            <!-- ─── Works Project 2 ─────────────────────────── -->
+            <div class="works-container">
+                <div class="banner">
+                    <img src="./images/Portfolio6.png" class="project-img" alt="project image">
+                </div>
+                <div class="project-content">
+                    <div class="project-title">
+                        <h2>Multi-Post <br> Stories</h2>
+                    </div>
+                    <div class="outline">
+                        <li class="banner-list">Canopy</li>
+                        <span class="dot"></span>
+                        <li class="banner-list dev">Back End Dev</li>
+                        <span class="dot"></span>
+                        <li class="banner-list">2015</li>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="tags">
+                        <ul>
+                            <li class="skill">html</li>
+                            <li class="skill">css</li>
+                            <li class="skill">javascript</li>
+                        </ul>
+                    </div>
+                    <div class="action">
+                        <button class="btn">See Project</button>
```

Comment:
- [ ] Every HTML button should be given the type property. Please fix this for all the buttons on your projects.